libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -23,7 +23,8 @@ import { HomeRoutingModule } from './home.routing';
     LayoutModule,
     NgOptimizedImage,
     CardsModule,
-    HomeRoutingModule
+    HomeRoutingModule,
+    SocialImpactModule
```

Comment:
This is not allowed, pages cannot import other pages. May cause circular dependency later. Instead use a shared lib like features/components/banners